src/index.html

```diff
@@ -22,6 +22,495 @@
+              class="card__button-link"
+            >
+              Buy
+            </a>
+          </div>
+        </div>
+      </div>
+
+      <div class="grid-item">
+        <div
+          class="card"
+          data-qa="card-hover"
+        >
+          <div class="card__top">
+            <img
+              src="./images/imac.jpeg"
+              class="card__picture"
+              alt="imac-picture"
+            />
+          </div>
+
+          <div class="card__product-info">
+            <h1 class="card__product-name">
+              APPLE A1419 iMac 27" Retina
+              <br />
+              5K Monoblock (MNED2UA/A)
+            </h1>
+
+            <p class="card__product-code">Product code: 195434</p>
+
+            <div class="card__reviev">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p>Reviews: 5</p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-a">Price:</p>
+
+              <p class="card__price-b">$2,199</p>
+            </div>
+
+            <a
+              href="#"
+              data-qa="hover"
+              class="card__button-link"
+            >
+              Buy
+            </a>
+          </div>
+        </div>
+      </div>
+      <div class="grid-item">
+        <div
+          class="card"
+          data-qa="card-hover"
+        >
+          <div class="card__top">
+            <img
+              src="./images/imac.jpeg"
+              class="card__picture"
+              alt="imac-picture"
+            />
+          </div>
+
+          <div class="card__product-info">
+            <h1 class="card__product-name">
+              APPLE A1419 iMac 27" Retina
+              <br />
+              5K Monoblock (MNED2UA/A)
+            </h1>
+
+            <p class="card__product-code">Product code: 195434</p>
+
+            <div class="card__reviev">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p>Reviews: 5</p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-a">Price:</p>
+
+              <p class="card__price-b">$2,199</p>
+            </div>
+
+            <a
+              href="#"
+              data-qa="hover"
+              class="card__button-link"
+            >
+              Buy
+            </a>
+          </div>
+        </div>
+      </div>
+
+      <div class="grid-item">
+        <div
+          class="card"
```

Comment:
The `data-qa` attribute should have the value 'card' instead of 'card-hover' to maintain consistency and correct behavior for QA automation.